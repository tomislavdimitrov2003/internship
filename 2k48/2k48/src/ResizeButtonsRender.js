export function ResizeButtons (props) {
    return (
        <div>
            <button type="button" onClick={() => props.setCurrentSize(4)}>4x4</button>
            <button type="button" onClick={() => props.setCurrentSize(5)}>5x5</button>
            <button type="button" onClick={() => props.setCurrentSize(6)}>6x6</button>
        </div>
    );
}