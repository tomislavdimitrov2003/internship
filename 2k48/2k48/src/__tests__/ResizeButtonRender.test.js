import { ResizeButtons } from "../ResizeButtonsRender"
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import React from 'react';


it('renders resize buttons correctly', () => {
    const wrapper = shallow(<ResizeButtons/>);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
});