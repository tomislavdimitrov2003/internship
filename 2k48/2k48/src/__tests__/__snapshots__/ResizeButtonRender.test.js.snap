// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders resize buttons correctly 1`] = `
<div>
  <button
    onClick={[Function]}
    type="button"
  >
    4x4
  </button>
  <button
    onClick={[Function]}
    type="button"
  >
    5x5
  </button>
  <button
    onClick={[Function]}
    type="button"
  >
    6x6
  </button>
</div>
`;
